apps/opik-backend/src/test/java/com/comet/opik/infrastructure/http/cors/CorsDisabledE2ETest.java

```diff
@@ -0,0 +1,73 @@
+package com.comet.opik.infrastructure.http.cors;
+
+import com.comet.opik.api.resources.utils.ClickHouseContainerUtils;
+import com.comet.opik.api.resources.utils.ClientSupportUtils;
+import com.comet.opik.api.resources.utils.MySQLContainerUtils;
+import com.comet.opik.api.resources.utils.RedisContainerUtils;
+import com.comet.opik.api.resources.utils.TestDropwizardAppExtensionUtils;
+import com.redis.testcontainers.RedisContainer;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.testcontainers.clickhouse.ClickHouseContainer;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import ru.vyarus.dropwizard.guice.test.ClientSupport;
+import ru.vyarus.dropwizard.guice.test.jupiter.ext.TestDropwizardAppExtension;
+
+import static com.comet.opik.api.resources.utils.ClickHouseContainerUtils.DATABASE_NAME;
+
+@Testcontainers(parallel = true)
```

Comment:
I'm just not sure about this one because we reuse containers. Although I'm not sure, it could lead to more containers being created. Have you tested it?

Reply:
Okay, I just checked, and this option has no effect.  Actually, we need to remove those flags as we no longer use the Junit life cycle due to the application context registry (`@RegisterExtension`).  In order to correctly use the parallel start, we need to use `Startables.deepStart`. 

Here are some links to explain it a bit better: 
- https://java.testcontainers.org/features/advanced_options/ 
- https://testcontainers.com/guides/testcontainers-container-lifecycle/

Away, you are good to go. We can remove those later.